corehq/couchapps/last_modified/views/by_last_modified/map.js

```diff
@@ -1,5 +1,7 @@
 function(doc){
     if (doc.doc_type == "Group") {
         emit(doc.last_modified, null);
+    } else if (doc.doc_type == "Domain") {
+        emit(doc.last_modified, null);
```

Comment:
(i realize this might be working right now because they're in different dbs, but i don't think we want to rely on that long term)